src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -217,13 +219,15 @@ public async Task<IActionResult> SetState([FromBody] string state)
         [HttpPost]
         [Authorize(AuthenticationSchemes = AuthLevelAuthenticationDefaults.AuthenticationScheme)]
         [Route("runtime/webhooks/{name}/{*extra}")]
+        [RequiresRunningHost]
```

Comment:
This is the fix. Requests will now queue, waiting for the host to be fully initialized